client/src/components/PizzaPlaceShow.js

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from "react"
+
+const PizzaPlaceShow = props => {
+    const [pizzaPlace, setPizzaPlace] = useState({
+        name: "",
+        address: "",
+        phoneNumber: "",
+        website: "",
+        hours: "",
+        imageUrl: ""
```

Comment:
for a show page we don't need to specify all the properties in state - we can, but it just means that we have to remember to update here everytime properties are added of removed from a pizzaPlace. Instead this initial state can be defined as an empty object